### PR TITLE
As an user, I can view information for a role in Pulp

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #3179
https://pulp.plan.io/issues/3179

### CODE REVIEW

File: pulp_ansible/app/models.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+from django.db import models
+
+from pulpcore.plugin.models import Content, Importer, Publisher
+
+
+log = getLogger(__name__)
+
+
+class AnsibleRoleContent(Content):
```
Comment (daviddavis) [inline pulp_ansible/app/models.py]: Should we call this `AnsibleRole`?

Comment (bmbouter) [reply pulp_ansible/app/models.py]: Yes I think so.

File: pulp_ansible/app/models.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    class Meta:
+        unique_together = (
+            'namespace',
+            'name',
+            'version'
+        )
+
+
+class AnsiblePublisher(Publisher):
```
Comment (bmbouter) [inline pulp_ansible/app/models.py]: Will this publisher only handle `AnsibleRole` stuff or will it handle other types in the future as well like `TheNextAnsibleThing`? If it's always-and-forever `AnsibleRole`, should this be called `AnsibleRolePublisher`?

File: pulp_ansible/app/models.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+from pulpcore.plugin.models import Content, Importer, Publisher
+
+
+log = getLogger(__name__)
+
+
+class AnsibleRole(Content):
+    """
+    A content type respresenting an ansible role
```
Comment (bmbouter) [inline pulp_ansible/app/models.py]: s/ansible role/Ansible Role/

File: pulp_ansible/app/models.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+class AnsiblePublisher(Publisher):
+    """
+    A Publisher for Ansible content.
+    """
+
+    TYPE = 'ansible'
+
+
+class AnsibleImporter(Importer):
```
Comment (bmbouter) [inline pulp_ansible/app/models.py]: Similar question here as with `AnsiblePublisher`.

**Comment by bmbouter** [comment]
I was able to install the plugin and makemigrations for it! It did make migrations correctly for the 3 models.

File: pulp_ansible/app/models.py (hunk 2 above)
Comment (daviddavis) [reply pulp_ansible/app/models.py]: My assumption (at least for now) was that we'd want to publish all Ansible content in a repo together (not just roles).

Comment (bmbouter) [reply pulp_ansible/app/models.py]: Great then this is the right name. Ack.

**Pull request merged**